centaur/src/main/resources/standardTestCases/wdl_draft3/struct_localize_delocalize/struct_localize_delocalize.wdl

```diff
@@ -0,0 +1,51 @@
+version draft-3
+
+struct FileContainer {
+  File bam
+  File index
+}
+
+workflow struct_localize_delocalize {
+  call struct_delocalize
+  call struct_localize { input: container = struct_delocalize.fc }
+
+  scatter(s in struct_localize.found_files) {
```

Comment:
highlighting's not right

Reply:
Huh, that’s the second time I’ve seen this recently. Either something changed or I just haven’t submitted much WDL recently...